lnwire/neighbor_rst.go

```diff
@@ -21,15 +21,26 @@ func (msg *NeighborRstMessage) Command() uint32 {
 }
 
 func (msg *NeighborRstMessage) Encode(w io.Writer, pver uint32) error {
-	return nil
+	_, err := w.Write([]byte("NeighborRstMessage"))
```

Comment:
Mostly for debugging purposes. It allows message type determination just by looking on message payload.